usaspending_api/api_contracts/contracts/v2/download/awards.md

```diff
@@ -133,14 +133,13 @@ This route sends a request to the backend to begin generating a zipfile of award
 ## Filter Objects
 
 ### Filters (object)
++ `agencies` (optional, array[Agency], fixed-type)
 + `award_amounts` (optional, array[AwardAmount], fixed-type)
 + `award_ids` (optional, array[string])
     Award IDs surrounded by double quotes (e.g. `"SPE30018FLJFN"`) will perform exact matches as opposed to the default, fuzzier full text matches.  Useful for Award IDs that contain spaces or other word delimiters.
 + `award_type_codes` (optional, array[string])
-+ `agencies` (optional, array[Agency], fixed-type)
 + `contract_pricing_type_codes` (optional, array[string])
-+ `transaction_keyword_search` (optional, string)
-    Filter awards by keywords in the award's transactions.
++ `def_codes` (optional, array[DEFC], fixed-type)
```

Comment:
If the `DEFC` type isn't defined in the same file then dredd won't be able to find it and it won't be considered a valid type.

You'll need to define a `DEFC` type within the API contract. You can use this [spending_by_geography](https://github.com/fedspendingtransparency/usaspending-api/blob/master/usaspending_api/api_contracts/contracts/v2/disaster/spending_by_geography.md) contract as an example and to pull the DEFC object from.